src/PKSim.Core/Model/ParameterMetaData.cs

```diff
@@ -9,6 +9,8 @@ public class ParameterMetaData : ParameterInfo
       public string ContainerType { get; set; }
       public string ContainerName { get; set; }
       public ParameterBuildMode BuildMode { get; set; }
+      public int? ValueOriginId { get; set; }
+      public bool IsInput { get; set; }
 
```

Comment:
was required to use 7.3 database with 7.2. Not used in 7.2

Reply:
Can you add a comments that those will be removed / merged with 7.3?